src/utils.ts

```diff
@@ -53,12 +53,14 @@ export const runIframe = (
       }
     };
 
+    let iframeEventHandler;
     const timeoutSetTimeoutId = setTimeout(() => {
       rej(TIMEOUT_ERROR);
       removeIframe();
+      window.removeEventListener('message', iframeEventHandler, false);
```

Comment:
seems like this could just be in `removeIframe` both here and in the handler and it'd be slightly simpler.